src/jobs/test.yml

```diff
@@ -0,0 +1,53 @@
+description: |
+  This job will automate running a set of backend/frontend tests.
+
+parameters:
+  # --------------------------------------------------------------------------------
+  # REQUIRED PARAMETERS
+  # You must provide a value for each of these parameters.
+  # ---------------------------------------------------------------------------------
+  language:
+    type: enum
+    enum: ["node", "go"]
+    description: |
+      The language the project is using.
+
+  # ---------------------------------------------------------------------------------
+  # OPTIONAL PARAMETERS
+  # These parameters have reasonable defaults.
+  # ---------------------------------------------------------------------------------
+  resource-class:
+    type: string
+    default: "medium"
```

Comment:
What are the other options for this?

Reply:
I just added. The performance plan that we have supports:
<img width="502" alt="Screen Shot 2022-12-30 at 1 04 11 PM" src="https://user-images.githubusercontent.com/70287489/210099854-231c8bc0-8143-4158-a845-71b7aafa54ee.png">